Extensions/SystemInfo/systeminfotools.js

```diff
@@ -36,4 +36,13 @@ gdjs.evtTools.systemInfo.isMobile = function() {
  */
 gdjs.evtTools.systemInfo.isWebGLSupported = function(runtimeScene) {
 	return runtimeScene.getGame().getRenderer().isWebGLSupported();
-};
\ No newline at end of file
+};
+
+/**
+ * Checks in runtimeGame for the additional isPreview spec.
```

Comment:
I think I already made this comment: you need to comment to give **additional** descriptions, i.e: **extra** context. But not to describe the code. Someone can always take a look at the code if they want to know what the code is doing.

`Check if the game is running as a preview, launched from an editor.`